### DISCUSSION TITLE
Ability to make different colors in theme for argument vs other variables?

### DISCUSSION THREAD

**Discussion by Object905** [zed-industries/zed#17532] [Feature Requests]
Hi! In VSCode in monokai theme I'm used to color for arguments being different from other variables. Any chance this somehow might be achieved in zed?
Looking in theming `syntax.variable.special` is handled for `self` in python, but I was unable to find all available config keys for `syntax` in [schema](https://zed.dev/schema/themes/v0.1.0.json).
Is this already supported? Should I create an issue for this?

VSCode:
![image](https://github.com/user-attachments/assets/961d79d4-5b15-4efb-9dc6-82c4291eb9ba)
zed:
![image](https://github.com/user-attachments/assets/a7d5a13b-e274-4123-ac62-0bf71f32405b)

**Comment by pbelskiy** [comment]
Agree, also no way to configure color of import of Zedokai, it should be white

<img width="295" alt="Screenshot 2024-09-15 at 00 18 27" src="https://github.com/user-attachments/assets/77c6f62e-fc47-406c-b296-7a5d4b4633f5">

**Comment by HaraldWik** [comment]
Is there any way of doing this?

**Comment by elisiariocouto** [comment]
Most of the capture groups required to enable this are implemented in #21454.
I’m waiting for the merge to publish a theme based on the original Sublime Text 3 Monokai theme
